dist/modules/sound.js

```diff
@@ -446,6 +446,7 @@
 		Debug,
 		SoundContext,
 		SoundInstance,
+		CordovaAudioPlugin = include('createjs.CordovaAudioPlugin'),
```

Comment:
The build integrations are failing on this - would that be solved by the corresponding SoundJS PR?